Sources/ApolloTestSupport/XCTestCase+Helpers.swift

```diff
@@ -0,0 +1,32 @@
+import XCTest
+
+public extension XCTestExpectation {
+  var numberOfFulfillments: Int {
+    value(forKey: "numberOfFulfillments") as! Int
```

Comment:
Yep, unfortunately...

Reply:
🤮 all right. We may want to make this a `.apollo.numberOfFulfillments` extension in case they get off their butts and expose this publicly next year though

Reply:
Yep, better safe than sorry I guess.